modules/cloudsql-instance/main.tf

```diff
@@ -234,8 +234,8 @@ resource "google_sql_user" "users" {
   type     = each.value.type
 }
 
-resource "google_sql_ssl_cert" "postgres_client_certificates" {
-  for_each    = var.postgres_client_certificates != null ? toset(var.postgres_client_certificates) : toset([])
+resource "google_sql_ssl_cert" "client_certificates" {
```

Comment:
can you add a moved block for this? Otherwise you'll force recreation for existing users